app/App.jsx

```diff
@@ -335,9 +352,20 @@ class App extends React.Component {
     //     this.refs.notificationSystem.addNotification(params);
     // }
 
+    onErrorModule() {
+        this.setState({errorModule: false});
+    }
+
     render() {
         let {incognito, incognitoWarningDismissed} = this.state;
-        let {walletMode, theme, location, match, ...others} = this.props;
+        let {
+            walletMode,
+            theme,
+            location,
+            match,
+            isErrorCaught,
```

Comment:
Is this even set within App.jsx?

Reply:
It's set in AppInit's componentDidCatch.